packages/connect-iframe/tsconfig.json

```diff
@@ -2,7 +2,8 @@
     "extends": "../../tsconfig.json",
     "compilerOptions": {
         "outDir": "./libDev",
-        "types": ["web", "w3c-web-usb", "jest"]
+        "types": ["web", "w3c-web-usb", "jest"],
+        "isolatedModules": false
```

Comment:
is it needed? I tried removing this and everything worked the same, so maybe it is some leftover?

Reply:
Yes, it is possible, seams to work without it now. Removed `"isolatedModules": false`.